tests/gegl/Makefile.am

```diff
@@ -29,8 +29,8 @@ endif
 LDADD = \
 	$(DEPS)								\
 	$(GEGL_LIBS)						\
+	$(top_builddir)/tests/libmypaint-tests.a \
 	$(top_builddir)/libmypaint.la		\
-	$(top_builddir)/gegl/libmypaint-gegl.la	\
-	$(top_builddir)/tests/libmypaint-tests.a
+	$(top_builddir)/gegl/libmypaint-gegl.la
```

Comment:
Yes. Try adding `-Wl,--no-undefined` to your `LDFLAGS’ and test with and without this patch. :-)